react/components/ProductDescription/index.js

```diff
@@ -11,21 +11,29 @@ import styles from './styles.css'
  * Product Description Component.
  * Render the description of a product
  */
-const ProductDescription = ({ description }) => {
+const ProductDescription = ({ description, showShowMoreButton }) => {
   if (!description) {
     return null
   }
 
+  const descriptionParsed = useMemo(() => HtmlParser(description), [description])
+
   return (
     <div className={styles.productDescriptionContainer}>
       <FormattedMessage id="store/product-description.title">
         {txt => <h2 className="t-heading-5 mb5 mt0">{txt}</h2>}
       </FormattedMessage>
 
       <div className="c-muted-1">
+        {showShowMoreButton ? (
         <GradientCollapse collapseHeight={220}>
-          {HtmlParser(description)}
+          {descriptionParsed}
         </GradientCollapse>
+        ) : (
+          <Fragment>
+            {descriptionParsed}
+          </Fragment>
+        )}
```

Comment:
You dont need a Fragment here, just: `{showShowMoreButton ? (<GradientCollapse...) : descriptionParsed}` is fine